openassessment/templates/openassessmentblock/message/oa_message_complete.html

```diff
@@ -4,12 +4,8 @@
     <h3 class="message__title">{% trans "You Have Completed This Assignment" %} </h3>
     <div class="message__content">
         <p>
-            {% if waiting == 'peer' %}
-                <p>{% trans "Your grade will be available when your peers have completed their assessments of your response." %}</p>
-            {% elif waiting == 'example-based' %}
-                <p>{% trans "Your grade will be available when the example-based assessment of your response is complete." %}</p>
-            {% elif waiting == 'all' %}
-                <p>{% trans "Your grade will be available when your peers have completed their assessments of your response and the example-based assessment of your response is complete." %}</p>
+            {% if waiting %}
+                <p>{% trans "Not all assessment steps have been completed for your response. After all assessment steps are complete, you will see feedback from everyone who assessed your response, and you will receive your final grade." %}</p>
```

Comment:
@cahrens How about: "Your final grade will be available when all assessment steps for your response have been completed."